torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -2041,6 +2040,17 @@ void ONNXShapeTypeInference(
       GRAPH_DEBUG(
           "ONNX graph after shape inference: ", prettyPrint(*model_proto));
     }
+  } else if (n->output()
+                 ->type()
+                 ->cast<TensorType>()
+                 ->sizes()
+                 .concrete_sizes()
+                 .has_value()) {
```

Comment:
@BowenBao Is there a more elegant way to approach what I wanted to do here. Basically, a non-ONNX op which has shape/type information should be get in this block.